app/views/profiles/show.html.erb

```diff
@@ -62,6 +62,15 @@
               <%= @profile.city %></span>
             </p>
           <% end %>
+
+          <% if @profile.country.present? %>
+            <h2 class="delta sans-serif mb-">
+              <%= t(:country, scope: 'profiles.profile') %>
+            </h2>
+            <p class="bb pb">
+              <%= @profile.country %></span>
```

Comment:
could you change the displayed information into the country name of the corresponding locale? Currently if I chose "Germany" as a country in the profile show action "DE" is displayed. It would be cool if I would see "Deutschland" in the german locale, "Germany" in the english and "Alemanha" (in the future Portuguese locale)